examples/development/web2/index.js

```diff
@@ -1,12 +1,13 @@
 var CodeMirror = require("codemirror");
-require("./node_modules/codemirror/lib/codemirror.css");
+require("css!./node_modules/codemirror/lib/codemirror.css");
+require("css!./node_modules/jupyter-js-widgets/static/widgets/css/widgets.min.css");
 require("./node_modules/codemirror/mode/python/python");
 
 var WidgetManager = require("./manager").WidgetManager;
 
 document.addEventListener("DOMContentLoaded", function(event) {
 
-    var code = require("./widget_code.json").join("\n");
+    var code = require("json!./widget_code.json").join("\n");
```

Comment:
Thanks, good comments. I definitely prefer that the webpack config shows how to load rather than changing the `require` syntax.